### ISSUE TITLE
Quesions about the learning rate for the 4GPUS.

### ISSUE THREAD

**Issue by justimyhxu** [facebookresearch/TimeSformer#59]
I have checked your configs for 4GPUS. The total batch size is $NUM_GPUS$*$TRAIN.BATCH_SIZE$ = 16.
While the total batch size of the official config for 8GPUS is 8*8=64， the learning rate is also as same as the 4GPUS.
Following the linear scaling rule of the learning rate of SGD, the 4GPUS learning should be 0.005*16/64 (assume the 0.005 is set for the 8GPUS). I am wondering that why the learning rate for 4GPUS is still 0.005?

**Comment by gberta** [comment]
Based on my preliminary experiments, scaling the learning rate didn't change the performance that much. Therefore, I didn't use the linear scaling rule, and instead set the learning rate to 0.005 regardless of the number of GPUs.  I didn't try this with 4 GPUs though. Therefore, you could try both variants (scaling vs not scaling) and see which one work better. Let me know if scaling the learning rate improves the performance. It would be useful to know.

**Comment by justimyhxu** [comment]
I have implemented timesformer in our codebase. I do not conduct the experiments with 4GPUs. Our training setting is 32 videos on 8 GPUs. The learning rate is scaled to 0.0025, and the result is 78.15%, a little better than the results reported in this repo. Maybe the linear-scaling rule always works for SGD.

**Comment by liyz15** [comment]
It seems that with the default config  `GLOBAL_BATCH_SIZE = 64`, the effective batch size will always be 64?

https://github.com/facebookresearch/TimeSformer/blob/a7b3ee2560675a0b9fbae40008f504c3243e29ac/tools/train_net.py#L102-L116
